src/patients/new/DuplicateNewPatientModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { Modal, Alert } from '@hospitalrun/components'
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import useTranslator from '../../shared/hooks/useTranslator'
+import Patient from '../../shared/model/Patient'
+
+interface Props {
+  duplicatePatient?: Patient
+  show: boolean
+  toggle: () => void
+  onCloseButtonClick: () => void
+  onContinueButtonClick: () => void
+}
+
+const DuplicateNewPatientModal = (props: Props) => {
+  const { t } = useTranslator()
+  const { duplicatePatient, show, toggle, onCloseButtonClick, onContinueButtonClick } = props
+
+  const alertMessage =
+    'Patient with matching information found in database. Are you sure you want to create this patient?'
+
+  const body = (
+    <>
+      <Alert color="danger" title={t('Warning!')} message={t(alertMessage)} />
```

Comment:
Thoughts on having this as a warning color rather than danger?

Reply:
I think it makes sense. It's more of a warning message than an error message, and it still stands out.